frontend/packages/console-app/locales/en/console-app.json

```diff
@@ -28,18 +28,17 @@
   "Storage Class": "Storage Class",
   "Requested capacity": "Requested capacity",
   "Used capacity": "Used capacity",
-  "Access mode": "Access mode",
   "Volume mode": "Volume mode",
   "Save": "Save",
   "Restore as new PVC": "Restore as new PVC",
   "When restore action for snapshot <1>{{snapshotName}}</1> is finished a new crash-consistent PVC copy will be created.": "When restore action for snapshot <1>{{snapshotName}}</1> is finished a new crash-consistent PVC copy will be created.",
-  "Volume Mode": "Volume Mode",
   "Size should be equal or greater than the restore size of snapshot": "Size should be equal or greater than the restore size of snapshot",
   "{{resourceKind}} details": "{{resourceKind}} details",
   "Created at": "Created at",
   "Status": "Status",
   "API version": "API version",
   "Restore": "Restore",
+  "Only {allowedVolumeModes[0]} volume mode is available for {storageClass} with {accessMode} access mode": "Only {allowedVolumeModes[0]} volume mode is available for {storageClass} with {accessMode} access mode",
```

Comment:
The first variable going to be a little confusing for the translators. It would be better if we can avoid in-lining the logic in the message.

Reply:
ack